webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationService.java

```diff
@@ -46,6 +47,7 @@
     private static final String ValidationFormat = "Row: %1$d Failure: %2$s\n";
     private static final String MoreFailuresFormat = "With %1$d additional validation failures\n";
     private static final int MaxFailureMessages = 100;
+    static final int MaxStudentsPerGroup = 200;
```

Comment:
May not be an immediate need but I am wondering if this should be configurable?
Probably not for phase 1 anyway because this is the agreed hard limit.